plugins/doc_fragments/pulp.py

```diff
@@ -42,6 +42,11 @@ class ModuleDocFragment(object):
       - It is recommended to use this once with the M(pulp.squeezer.status) module at the beginning of the playbook.
     type: bool
     default: false
+  pulp_timeout:
+    description:
+      - timeout value for connecting to pulp api
```

Comment:
This is not what timeout does.

Reply:
`time in seconds to wait for successful connection to pulp api` - would that be a better description?

Reply:
No, you are trying to make this parameter available to the not yet rewritten modules:
https://github.com/pulp/squeezer/blob/develop/plugins/doc_fragments/pulp.py#L63

Reply:
I am not sure if I follow. 
I suppose what you're saying is that you're rewriting modules to use pulp_glue which already has this timeout?
As far as I can see, at least the rpm related modules are still using this openapi.py and it lacks this setting, which causes me some issues that I described in https://github.com/pulp/squeezer/issues/163.
Can you explain what I am supposed to do to get this merged?

Reply:
Yes, basically.
You should use the same name and the same description at the very least.